### PR TITLE
FEATURE: Add option to include all categories

### PR DESCRIPTION
**Pull request by samamorgan**
This commit adds the option to use a wildcard "*" to enable ticket creation for all categories.

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ en:
   site_settings:
     zendesk_url: Your Zendesk URL. e.g. https://YOUR_SUBDOMAIN.zendesk.com/api/v2
     zendesk_enabled: Enable Zendesk plugin
-    zendesk_enabled_categories: List of categories that will autogenerate Zendesk tickets
+    zendesk_all_categories: Create tickets for all categories? If enabled, `zendesk_enabled_categories` are ignored.
```
Comment (techAPJ) [inline config/locales/client.en.yml:13]: I recommend changing the setting name to `zendesk_enable_all_categories`.

Also, let's replace the question mark `?` with period `.` in setting description for the sake of consistency.

**Review by techAPJ — CHANGES_REQUESTED** [review]
Added a minor feedback. Thanks for this PR.

**Comment by samamorgan** [comment]
@techAPJ Great suggestions! Done.

**Review by techAPJ — APPROVED** [review]
Looks good. 👍🏼

**Comment by techAPJ** [comment]
Thanks @samamorgan. ❤️

**Pull request merged**